test/test_ci_container.rb

```diff
@@ -27,23 +28,36 @@ def setup
       config.default_cassette_options = {
         match_requests_on:  [:method, :uri, :body]
       }
+      config.filter_sensitive_data('<%= Dir.pwd %>', :erb_pwd) { Dir.pwd }
     end
 
-    # Chdir to root, as Containment will set the working dir to PWD and this
-    # is slightly unwanted for tmpdir tests.
-    Dir.chdir('/')
-
     @job_name = self.class.to_s
     @image = 'ubuntu:15.04'
-    VCR.turned_off { cleanup_container }
+    VCR.turned_off do
+      cleanup_container
+    end
   end
 
   def teardown
     VCR.turned_off { cleanup_container }
   end
 
+  # FIXME: Make vcr_it a common method for tests
```

Comment:
I only just realized, can't be made common, one might want to do some sort of test setup for recording cassettes, so I guess I should drop the fixme